packages/site/src/components/PageBlock.tsx

```diff
@@ -65,15 +65,14 @@ export const PageBlock = ({ structure }: PageBlockProps) => {
         {structure.useCategories && (
           <div
             style={{
-              width: 305,
+              width: "240px",
```

Comment:
Just a slightly more snug SegmentedControl with more spacing between it and the collection